terraform/modules/ecs/main.tf

```
@@ -34,7 +34,7 @@ locals {
       FARGATE_EXECUTOR_SECURITY_GROUPS = module.fargate_sg.security_group_id,
       SINGLE_PLAYER                    = var.polytomic_single_player,
       WORKOS_API_KEY                   = var.polytomic_workos_api_key,
-      REACT_APP_WORKOS_CLIENT_ID       = var.polytomic_react_app_workos_client_id,
+      WORKOS_CLIENT_ID                 = var.polytomic_react_app_workos_client_id,
```

```suggestion
      WORKOS_CLIENT_ID                 = var.polytomic_workos_client_id,
```

We'll want to use the newly named variable for the value here

---

Fixed with: [68d0c60](https://github.com/polytomic/on-premises/pull/12/commits/68d0c60e2d54b4f2f9cf7bfcaf90ef6fdda0810c)